launchpad/pkg/flagd.go

```diff
@@ -154,19 +154,27 @@ func stopFlagDWithoutLock() error {
 			return fmt.Errorf("failed to stop flagd: %v", err)
 		}
 		flagdCmd = nil
+		fmt.Println("flagd stopped")
 	}
 	return nil
 }
 
-func monitorOutput(pipe io.ReadCloser, ready chan bool) {
+func monitorOutput(pipe io.ReadCloser, ready chan bool, info string) {
```

Comment:
```suggestion
func monitorOutput(pipe io.ReadCloser, ready chan bool, stream string) {
```

This should probably be called stream.